### PR TITLE
Ensure popup text is aligned left

### PR DESCRIPTION
**Pull request by saitho**
When hovering over the pull request number, a popup opens with more details on the pull request.
In branch list, the contents are aligned to the right since the original link also is.

https://try.gitea.io/saitho/test/branches
Screenshot from 2021-10-16 16-47-43

Setting a `text-align` on popup solves that.

### CODE REVIEW

**Comment by wxiaoguang** [comment]
No, you shouldn't overwrite the global style. Please check what's wrong with that element.

image

**Comment by silverwind** [comment]
Fomantic UI is using `text-align: right` to align non-text content and that rule unfortunately cascades into the popup content. A arguably better solution would be to change the column layout to use flexbox with `justify-content: space-between`.

File: web_src/less/_base.less [hunk 1]
```diff
@@ -1782,6 +1782,7 @@ a.ui.basic.label:hover {
   background-color: var(--color-body);
   color: var(--color-secondary-dark-6);
   border-color: var(--color-secondary);
+  text-align: left;
```
Comment (silverwind) [inline web_src/less/_base.less]:
```suggestion
  text-align: left; /* fix alignment of PR popup in branches table */
```

**Review by silverwind — APPROVED** [review]
Happy to take this as a temporary fix with that comment until we can refactor branch table to flexbox.

File: web_src/less/_base.less (hunk 1 above)
Comment (wxiaoguang) [reply web_src/less/_base.less]: I am against it. Do not pollute global styles anymore.

Comment (silverwind) [reply web_src/less/_base.less]: Could scope it:

```css
/* fix alignment of PR popup in branches table */
.branches table .ui.popup {
  text-align: left;
}

**Comment by saitho** [comment]
I applied the suggestions.

**Review by wxiaoguang — APPROVED** [review]

**Comment by zeripath** [comment]
make lgtm work

**Pull request merged**